packages/atlas/patches/find.gtstorage.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/cmake/features/GRIDTOOLS_STORAGE.cmake b/cmake/features/GRIDTOOLS_STORAGE.cmake
+index 9d2dbb7..ff82c17 100644
+--- a/cmake/features/GRIDTOOLS_STORAGE.cmake
++++ b/cmake/features/GRIDTOOLS_STORAGE.cmake
+@@ -9,17 +9,20 @@
+       set( THREADS_PREFER_PTHREAD_FLAG 1 )
+     endif()
+ 
+-find_package( GridTools QUIET 
+-    HINTS ${GridTools_ROOT}/lib/cmake
+-          $ENV{GridTools_ROOT}/lib/cmake
+-          ${CMAKE_PREFIX_PATH}/lib/cmake
+-          ${CMAKE_INSTALL_PREFIX}/lib/cmake
+-          ${GridTools_BINARY_DIR} )
+ ecbuild_add_option(
+   FEATURE GRIDTOOLS_STORAGE
+   DESCRIPTION "Arrays internally use GridTools storage layer"
+   CONDITION GridTools_FOUND )
+ 
++if(ENABLE_GRIDTOOLS_STORAGE)
++  find_package( GridTools QUIET 
++      HINTS ${GridTools_ROOT}/lib/cmake
++            $ENV{GridTools_ROOT}/lib/cmake
++            ${CMAKE_PREFIX_PATH}/lib/cmake
++           ${CMAKE_INSTALL_PREFIX}/lib/cmake
++            ${GridTools_BINARY_DIR} )
++endif(ENABLE_GRIDTOOLS_STORAGE)
++
```

Comment:
Can it be deleted if not needed?

Reply:
these were not commented out but rather put under an if. In general this find_package could be triggered if the option is activated